### ISSUE TITLE
Latest update unusable on (at least) Ubuntu 19.10 Server

### ISSUE THREAD

**Issue by Groofster** [bubuntux/nordvpn#66]
Latest version is not working at all on Ubuntu 19.10 Server, creating the following errors:

> 2020/03/13 15:54:38 adding route for subnet 172.21.0.0/16: executing 'ip route add' command: exit status 2: RTNETLINK answers: File exists
> 
> INFO: (nordlynx) 2020/03/13 15:54:38 Device closing
> INFO: (nordlynx) 2020/03/13 15:54:38 Interface closed
> 2020/03/13 15:54:38 [nordlynx] ip rule delete to 185.156.175.132
> We're having trouble configuring your firewall settings. If the issue persists, please contact our customer support.

This problem was reproduced on 2 separate Ubuntu 19.10 installations after a container update. No OS updates were done on the second machine to verify errors on first.

**Comment by bubuntux** [comment]
i just saw the same issue on my end, please fallback to prev version while i look at this

`bubuntux/nordvpn:3.6.1-1`

**Comment by Groofster** [comment]
Falling back worked like a charm. Thank you for the lighting fast response.

If I can help in any way, please let me know.

**Comment by bubuntux** [comment]
seems like the error goes away when removing 

`	[[ -n ${DOCKER_NET} ]]  && nordvpn whitelist add subnet $DOCKER_NET`

from the start_vpn.sh 
but not sure if that is the intereaction between containers will work, i can test later tonight, hopefully everything works after that change

**Comment by LexJackson** [comment]
Getting the same thing.

````
|INFO: (nordlynx) 2020/03/13 15:48:26 Attaching to interface
INFO: (nordlynx) 2020/03/13 15:48:26 Interface set up
/INFO: (nordlynx) 2020/03/13 15:48:26 Device started
2020/03/13 15:48:26 address :10.5.0.2/16
INFO: (nordlynx) 2020/03/13 15:48:26 Interface set down
2020/03/13 15:48:26 [nordlynx] ip -4 address add 10.5.0.2/16 dev nordlynx
ERROR: (nordlynx) 2020/03/13 15:48:26 peer(ruuf…dJCc) - Failed to send handshake initiation no bind
2020/03/13 15:48:26 [nordlynx] ip link set mtu 1420 up dev nordlynx
INFO: (nordlynx) 2020/03/13 15:48:26 Interface set up
2020/03/13 15:48:26 sysctl: setting key "net.ipv6.conf.all.disable_ipv6": Read-only file system
We could not disable IPv6. For more information, please check activity log.
2020/03/13 15:48:26 [nordlynx] ip rule add to 192.158.226.51 lookup main
-2020/03/13 15:48:26 adding route for subnet 192.168.32.0/20: executing 'ip route add' command: exit status 2: RTNETLINK answers: File exists
INFO: (nordlynx) 2020/03/13 15:48:26 Device closing
INFO: (nordlynx) 2020/03/13 15:48:26 Interface closed
2020/03/13 15:48:26 [nordlynx] ip rule delete to 192.158.226.51
We're having trouble configuring your firewall settings. If the issue persists, please contact our customer support.
````

**Comment by obbardc** [comment]
ahhh, with fallback version `bubuntux/nordvpn:3.6.1-1`

I get an error: `A new version of NordVPN is available! Please update the application.`

:-(

**Comment by bubuntux** [comment]
i just pushed a fix for the issues related to version 3.7 (latest) seems to be working fine on my end

**Comment by LexJackson** [comment]
Nice work and thank you!

**Comment by LexJackson** [comment]
@bubuntux I had to fail back to :3.6.1-1 as my containers couldn't talk to each other with the latest push.

**Comment by bubuntux** [comment]
> my containers couldn't talk to each other
can you elaborate ? how can i test that ? my setup worked just fine, but i do see that could be an issue with the latest change =/

it's very odd cuz nordvpn throw an error saying that the iprule already exist =/

**Comment by LexJackson** [comment]
No problem sorry for the light information. I have two docker compose stacks. Both use the NordVPN container and all other containers inside the 2 stacks are dependent on the VPN container executing first. When I reconfigured both stacks to use master, stack one couldn’t talk to stack two. And there was some intra stack communication issues between the containers inside of stack one. Happy to post my docker-compose files if you would like.

**Comment by bubuntux** [comment]
i think this a bug on Nordvpn application, i already reach out to them and filed a bug, hopefully they will fix it soon.

**Comment by Groofster** [comment]
I'm going completely mad here now. On my setup I get the following error in the log of the container:

> Technology is successfully set to 'NordLynx'.
> DNS is set to '1.0.0.1, 1.1.1.1, 8.8.4.4' successfully.
> Subnet 172.21.0.0/16 is whitelisted successfully.
> Subnet 10.1.1.0/24 is whitelisted successfully.
> Please enter your login details.
> Stdin/Stdout should be terminal

Older or newer versions, doesn't make any differnce. The same setup I admin for a friend on his Ubuntu Server is working perfectly......

Have to go back to the openvpn version for now but unfortunately that one doesn't honor my country settings. Not with the COUNTRY setup or with the CONNECT setting.

Hoping someone has seen this one......

**Comment by bubuntux** [comment]
I made this image to fulfill my needs, i'm gonna keep it public and you can use it at your own risk, if something doesn't work for you, sry but is up to you to fix it, unfortunately i don't have time to keep addressing ppls issues, from now on i'm only accepting pull requests.

**Comment by Groofster** [comment]
Thank you. Much respect for the time you already spend on this. Godspeed !

**Comment by bubuntux** [comment]
should be fixed with e0fa84966fbddc6053d72ccd4bb20d23a947234b
